tests/backwards_compatibility_tests/compatibility_test_logger.py

```diff
@@ -0,0 +1,7 @@
+import logging
+
+def get_logger(name):
```

Comment:
Is this logger used by test runner only?

Reply:
yes